napari/layers/base/base.py

```diff
@@ -458,7 +465,7 @@ def _mode_setter_helper(self, mode):
             callback_list.append(mode_dict[mode])
         self.cursor = self._cursor_modes[mode]
 
-        self.interactive = mode == self._modeclass.PAN_ZOOM
+        self.mouse_pan = mode == self._modeclass.PAN_ZOOM
```

Comment:
Next place that suggests that the `Mode` enum should be updated.

Reply:
See the first comment above on this.